image/local.go

```diff
@@ -134,6 +134,16 @@ func (l *local) Digest() (string, error) {
 	return parts[1], nil
 }
 
+func (l *local) CreatedAt() (time.Time, error) {
+	createdAtTime := l.Inspect.Created
+	createdTime, err := time.Parse(time.RFC3339Nano, createdAtTime)
+
+	if err != nil {
+		return time.Now(), err
```

Comment:
Return empty time here instead of now. No need to make an extra syscall for undefined values.